src/editor/Sidebar.js

```diff
@@ -156,13 +161,35 @@ const Sidebar = ( {
 				</>
 			) }
 			{ placement === 'inline' && (
-				<RangeControl
-					label={ __( 'Approximate Position (in percent)', 'newspack-popups' ) }
-					value={ trigger_scroll_progress }
-					onChange={ value => onMetaFieldChange( 'trigger_scroll_progress', value ) }
-					min={ 0 }
-					max={ 100 }
-				/>
+				<>
+					<SelectControl
+						label={ __( 'Trigger', 'newspack-popups' ) }
```

Comment:
The overlay prompts are "triggered", since they are initially hidden. Inline prompts are just inserted into content.

```suggestion
						label={ __( 'Insertion position', 'newspack-popups' ) }
```